start.sh

```diff
@@ -1,3 +1,48 @@
 #!/bin/sh
 
+if [[  -n "${LEAN_DB_PASSWORD_FILE}" ]]; then
+  LEAN_DB_PASSWORD=$(cat "${LEAN_DB_PASSWORD_FILE}")
+  export LEAN_DB_PASSWORD
+fi
+
+if [[  -n "${LEAN_EMAIL_SMTP_PASSWORD_FILE}" ]]; then
+  LEAN_EMAIL_SMTP_PASSWORD=$(cat "${LEAN_EMAIL_SMTP_PASSWORD_FILE}")
+  export LEAN_EMAIL_SMTP_PASSWORD
+fi
+
+if [[  -n "${LEAN_S3_SECRET_FILE}" ]]; then
+  LEAN_S3_SECRET=$(cat "${LEAN_S3_SECRET_FILE}")
+  export LEAN_S3_SECRET
+fi
+
+if [[  -n "${LEAN_SESSION_PASSWORD_FILE}" ]]; then
+  LEAN_SESSION_PASSWORD=$(cat "${LEAN_SESSION_PASSWORD_FILE}")
+  export LEAN_SESSION_PASSWORD
+fi
+
+if [[  -n "${LEAN_REDIS_PASSWORD_FILE}" ]]; then
+  LEAN_DB_PASSWORD=$(cat "${LEAN_REDIS_PASSWORD_FILE}")
```

Comment:
var should be set to the same value as the export line below, or it will not export properly

Reply:
thnx for spotting it!